packages/components/src/__future__/Select/_docs/Select.stories.tsx

```diff
@@ -170,25 +171,41 @@ export const FullWidth: Story = {
 export const PortalContainer: Story = {
   render: args => {
     const portalContainerId = "id--portal-container"
+
+    const [isOpen, setIsOpen] = React.useState(false)
+
+    const handleOpen = (): void => setIsOpen(true)
+    const handleClose = (): void => setIsOpen(false)
     return (
       <>
-        <div
-          id={portalContainerId}
-          className="flex gap-24 bg-gray-200 p-12 overflow-hidden h-[200px] relative"
-        >
-          <Select
+        <div className=" h-[500px] mb-24 block bg-gray-100 flex flex-col gap-16 justify-center items-center">
+          Page content
+          <button
+            type="button"
+            className="border border-gray-500"
+            onClick={handleOpen}
+          >
+            Open Modal
+          </button>
+          <ContextModal
             {...args}
-            label="Default"
-            selectedKey="batch-brew"
-            id="id--select-default"
-          />
-          <Select
-            {...args}
-            label="Inner portal"
-            selectedKey="batch-brew"
-            id="id--select-inner"
-            portalContainerId={portalContainerId}
-          />
+            isOpen={isOpen}
+            onConfirm={handleClose}
+            onDismiss={handleClose}
+            title="Select test"
+          >
+            <div
+              className="flex gap-24 bg-gray-200 p-12 h-[500px] relative"
```

Comment:
Remove the height on this container, since it doesn't actually showcase the solution this provides, which is that the listbox doesn't get chopped off and sits on top of the Modal.
Most of the styles can probably go aside from `relative` :)

Initially I was going to suggest moving the `id` and `position: relative` to `ContextModal`, but it seems the props aren't set up properly 😅 

<img width="878" alt="image" src="https://github.com/user-attachments/assets/6b0ccbd3-0ac4-41b4-9c1a-745b09c9a480">

Reply:
Ah gotcha, will do 👍